src/server/server.js

```diff
@@ -41,16 +41,22 @@ function sendHostJsFile(response, hostType) {
 }
 
 function streamAppHostHtml(request, response) {
-    // Always prepare before serving up app HTML file, so app is up-to-date, then create the app-host.js (if it is
-    // out-of-date) so it is ready when it is requested.
+    var filePath = path.join(config.platformRoot, url.parse(request.url).pathname);
```

Comment:
Doesn't look like it will; my changes are limited to the scope of `prepare.prepare()`